docs/dev/building-on-zksync/useful-address.md

```diff
@@ -67,11 +72,22 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
 
 ## Mainnet known tokens
 
+These are the tokens that are listed on our block explorer. Learn how to [add token](../developer-guides/bridging/bridging-asset.md#add-tokens-to-the-bridge) to our bridge.
+
 |    Symbol                     | Name          | L1 token address                          | L2 token address                         |
 | ----------------------------- | ------------- | ------------------------------------------| ------------------------------------------
 | COMBO                         | Furucombo     | [0xffffffff2ba8f66d4e51811c5190992176930278](https://etherscan.io/address/0xffffffff2ba8f66d4e51811c5190992176930278)|[0xc2B13Bb90E33F1E191b8aA8F44Ce11534D5698E3](https://explorer.zksync.io/address/0xc2B13Bb90E33F1E191b8aA8F44Ce11534D5698E3#contract)|
 | MUTE                          | Mute.io       | [0xa49d7499271ae71cd8ab9ac515e6694c755d400c](https://etherscan.io/address/0xa49d7499271ae71cd8ab9ac515e6694c755d400c)|[0x0e97C7a0F8B2C9885C8ac9fC6136e829CbC21d42](https://explorer.zksync.io/address/0x0e97C7a0F8B2C9885C8ac9fC6136e829CbC21d42)|
 | PERP                          | Perpetual     |  [0xbC396689893D065F41bc2C6EcbeE5e0085233447](https://etherscan.io/address/0xbC396689893D065F41bc2C6EcbeE5e0085233447)|[0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601](https://explorer.zksync.io/address/0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601#contract)|
-| USDC                          | USD Coin      | [0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48)|[0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4](https://explorer.zksync.io/address/0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4#contract)|
+| USDC                          | USD Coin      | [0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48](https://etherscan.io/address/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48)|[0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4](https://explorer.zksync.io/address/0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4#contract)|
```

Comment:
Should this be a verified contract link for USD Coin? I don't think it's correct.

Reply:
Is L1 correct?